test/models/user_test.rb

```diff
@@ -26,4 +26,22 @@ class UserTest < ActiveSupport::TestCase
     u.email = "bar@grinnell.edu"
     assert_equal "[bar]", u.display_name
   end
+
+  def create_valid_user(
+        name: "defaultName",
+        email: "default@grinnell.edu",
+        password: "GreatBigOldDefaultPassword1@!5")
+    u = User.new
+    u.name = name
+    u.email = email
+    u.password = password
+    return u
+  end
```

Comment:
Models accept a hash.  Rather than using this helper, you can just do `User.new(name: "foo", email: "bar", password: "123456")`

Reply:
Fair enough. Fixed